site/tryit/README.md

```diff
@@ -7,7 +7,6 @@ lang: en-US
 
 The dockerhub `sorintlab/agola-demo` is a demo image to easily test agola, it comes with a predefined configuration that uses an embedded etcd. Don't use it in production but build your own image or use the `sorintlab/agola` image
```

Comment:
It seems to me that `sorintlab/agola-demo`  should be `sorintlab/agolademo`.

https://hub.docker.com/r/sorintlab/agolademo